app/main/routes.py

```diff
@@ -64,6 +64,19 @@ def faq():
 
     return render_template('faq.html', title='CONP | FAQ', user=current_user, content=content)
 
+@main_bp.route('/result')
+def result():
+    """ Result Route
+
+        Route to lead to the result page
+
+        Args:
+            None
+
+        Returns:
+            rendered template for faq.html
+    """
+    return render_template('result.html', title='CONP | Result', user=current_user)
```

Comment:
see comment above about the name of the page. Although, now that we have the analytics page, it would be nice instead to show that analytics in the analytics page directly.

Reply:
Can you please let me know where is analytics page on the portal?

Reply:
https://portal.conp.ca/analytics